network-api/networkapi/wagtailpages/factory/buyersguide.py

```diff
@@ -222,9 +222,9 @@ def create_general_product_visual_regression_product(seed, pni_homepage):
         data_control_policy_is_bad=True,
         company_track_record='Needs Improvement',
         track_record_is_bad=True,
-        track_record_details='What kind of track record are we talking about?',
+        track_record_details='<p> What kind of track record are we talking about? </p>',
```

Comment:
I updated these two fields so Percy would not mark them as unstyled, since the CSS rules are now looking for a <p> element